custom_components/grocy/__init__.py

```diff
@@ -197,14 +200,14 @@ def __init__(self, hass, client):
         self.hass.data[DOMAIN_DATA][STOCK_NAME] = (
             await self.hass.async_add_executor_job(self.client.stock, [True]))
 
-    @Throttle(MIN_TIME_BETWEEN_UPDATES)
+
```

Comment:
this seems like an unnecessary blank line

Reply:
Can you remind me again what is the tools to check import and code (I remember p8p, like codefactor, but it's another one) ?
Please

Reply:
isort for imports and pep8 for code style i think ;)